### PR TITLE
problem: fetching github  username to UI display

### PR DESCRIPTION
**Pull request by madguy02**
solution: published users collection from lib/database, subscribed the data on normal/ user.js, created template on views, user.html, used it on menu.html

### CODE REVIEW

**Comment by gazhayes** [comment]
Was this in response to an issue in the issue tracker?

**Comment by madguy02** [comment]
As, mentioned in the discussion, facebook Username display was fixed, hence i tried with github, and the Username wasn't displaying, so i submitted a PR for it

**Comment by madguy02** [comment]
https://github.com/Blockrazor/blockrazor/issues/44

File: client/normal/user.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Template } from 'meteor/templating';
+import { users } from '../../lib/database/user.js';
+//var user = Meteor.subscribe('users');
+Meteor.subscribe('users');
+Template.user.helpers({
+		UserName() {
+	 	return  Meteor.user().services.github.username;
```
Comment (gazhayes) [inline client/normal/user.js]: This will fail for any user that didn't sign up using Github.

Comment (madguy02) [reply client/normal/user.js]: #44 i didn't try out, as for facebook,i thought it was resolved

Comment (madguy02) [reply client/normal/user.js]: ok cool, i will put up for facebook too

Comment (madguy02) [reply client/normal/user.js]: ok i will put up for facebook too

File: lib/database/user.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+if (Meteor.isServer) {
+	Meteor.publish('users', function() {
```
Comment (gazhayes) [inline lib/database/user.js]: This introduces a huge security problem.

**Comment by madguy02** [comment]
In order to configure facebook login,  the last part of the page is not
showing, where we can put the clientID and details

File: client/normal/user.js (hunk 1 above)
Comment (gazhayes) [reply client/normal/user.js]: Facebook is already solved.

Comment (madguy02) [reply client/normal/user.js]: Is it possible to change the code acocrdingly and submit another PR for github issue?

File: lib/database/user.js (hunk 2 above)
Comment (madguy02) [reply lib/database/user.js]: Yeah, i am publishing more amount of data, than it is required.

Comment (madguy02) [reply lib/database/user.js]: Will fix this!

File: client/normal/user.js (hunk 1 above)
Comment (gazhayes) [reply client/normal/user.js]: Well you could, but this is not at all the simplest approach to solving the issue in #44. In fact it doesn't even solve the issue at all unless you also go through the entire codebase and check every place where usernames are used and then edit those parts as well. The simplest approach to use is the one that was already used for facebook.

Comment (madguy02) [reply client/normal/user.js]: So, is it better to tackle another issue for now? and properly understand the codebase for this PR.

Comment (gazhayes) [reply client/normal/user.js]: This is already one of the most simple issues to solve which is why it's left open as an assessment. If you look at the discussion thread and pull requests associated with this issue you can see that only a few lines of code are needed if you take the right approach. You are welcome to continue working on this as an open source project to build up your experience and public profile, but what we are looking for right now in terms of _paid_ contributors are people who have bit more experience with Meteor.

**Pull request closed without merging**